modules/nw-networkpolicy-audit-disable.adoc

```diff
@@ -0,0 +1,45 @@
+[id="nw-networkpolicy-audit-disable_{context}"]
+= Disabling network policy audit logging
+
+As a cluster administrator, you can disable network policy audit logging for a namespace.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in to the cluster with a user with `cluster-admin` privileges.
+
+.Procedure
+
+. To disable network policy audit logging for a namespace, enter the following command:
++
+[source,terminal]
+----
+$ oc annotate namespace <namespace> k8s.ovn.org/acl-logging-
```

Comment:
To disable audit logging if it was enabled.
oc annotate --overwrite namespace <namespace> k8s.ovn.org/acl-logging={}

e.g. oc annotate --overwrite namespace arti-test1 k8s.ovn.org/acl-logging={}